Source/WTF/wtf/MathExtras.h

```diff
@@ -71,6 +71,9 @@ constexpr double sqrtOfTwoDouble = M_SQRT2;
 constexpr float sqrtOfTwoFloat = static_cast<float>(M_SQRT2);
 #endif
 
+constexpr double twoPiDouble = piDouble * 2.0;
```

Comment:
What's the benefit. I personally find the previous code slightly more readable and I'm not sure I see the benefit.